### PR TITLE
Microgalaxy lab updates

### PR DESCRIPTION
**Pull request by EngyNasr**
Changing to the new structure of [excalidraw](https://excalidraw.com/#room=14e15dbbc2aea9d3696e,TCWhNxBPAQIgcUPS4WGBfA), content still need to be filled

### CODE REVIEW

**Comment by bebatut** [comment]
Can you also add changes to `base.yml`? Thanks

File: communities/microgalaxy/lab/usegalaxy.eu/templates/footer.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <p>
       <span>
         <i class="fab fa-github"></i>
-        <a href="https://github.com/galaxyproject/galaxy_codex/tree/subdomain_unification/subdomains/singlecell">Edit this page on GitHub</a>
+        <a href="https://github.com/galaxyproject/galaxy_codex/tree/subdomain_unification/subdomains/microgalaxy">Edit this page on GitHub</a>
```
Comment (bebatut) [inline communities/microgalaxy/lab/usegalaxy.eu/templates/footer.html]: Have you tried this link? The branch should be `main` and the path is wrong `communities/microgalaxy/lab`

**Comment by paulzierep** [comment]
Some questions:
For `Data import and preparation` we got tools / training / workflows ... not clear to me why we need this, we already got sections for tools / training / workflows ... 

This function allows to automatically populate the tools table: https://github.com/paulzierep/galaxy_tool_extractor/blob/microgalaxy_lab/sources/bin/populate_labs.py

Maybe you can use it here already.
It could be adapted for the workflows as well. If there is way to Microbial isolates and Microbiome workflows.

**Comment by paulzierep** [comment]
Not sure where this comes from:
microGalaxy community support
[General Galaxy support](https://galaxyproject.org/support/)
[General Galaxy help forum](https://help.galaxyproject.org/)
[Galaxy Training Network Slack workspace](https://join.slack.com/t/gtnsmrgsbord/shared_invite/zt-2llyx6p8j-LmpEIsJu0t4MQkBctkN8qg)
[microGalaxy user community chat room (Same channel but using Matrix/Element)](https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im)
[Galaxy Training Community chat](https://matrix.to/#/#Galaxy-Training-Network_Lobby:gitter.im)
[Usegalaxy.org chat](https://matrix.to/#/#galaxyproject_Lobby:gitter.im)

But I guess it should also be added to the community table

File: communities/microgalaxy/lab/usegalaxy.eu/templates/footer.html (hunk 1 above)
Comment (EngyNasr) [reply communities/microgalaxy/lab/usegalaxy.eu/templates/footer.html]: yeah this was changed

**Pull request merged**